lib/v2/nasa/apod.js

```diff
@@ -0,0 +1,50 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+module.exports = async (ctx) => {
+    const rootUrl = 'https://apod.nasa.gov/apod/archivepix.html';
+    const response = await got({
+        method: 'get',
+        url: rootUrl,
+    });
+    const $ = cheerio.load(response.data);
+
+    const list = $('body > b > a')
+        .slice(0, 10)
```

Comment:
You may pass the `ctx.query.limit` common parameter here which allows users to adjust no. to items to fetch
related user docs: https://docs.rsshub.app/parameter.html